### PR TITLE
[217] Develop a Customer details list view page and Save Session in a Cookie

### PR DESCRIPTION
**Pull request by ainobs**
Screenshot (95)
## Task: `Name of given task`
- `Changes you made to given task`

### CODE REVIEW

File: submissions/victorAienobe/phpDatabase/saveCookie.php [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    //Querying the database to select all information available in the table
+    try {
+        $sql= "SELECT * FROM customer_details";
+        $result = $pdo->query($sql);
+        
+    // setting up the cookie to save the result gotten from the databse
+    setcookie("customer", json_encode($result), time()+7200, "/");
+    include_once('accessCookie.php');
+
+        if($result-> rowCount() > 0)  {
```
Comment (adaugochi) [inline submissions/victorAienobe/phpDatabase/saveCookie.php:14]: Your implementation still doesn't satisfy this `Ensure that you are not hitting the database each time there is a page reload as long as the cookie is still set or hasn't expired yet` criteria. What is having currently, if I reload the page, the `$result` is still coming from the SQL query

**Review by adaugochi — CHANGES_REQUESTED** [review]

File: submissions/victorAienobe/phpDatabase/accessCookie.php [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<?php
+
+    if(isset($_COOKIE["customer"])) {
```
Comment (adaugochi) [inline submissions/victorAienobe/phpDatabase/accessCookie.php:3]: This file is unnecessary

**Review by adaugochi — CHANGES_REQUESTED** [review]

**Comment by ainobs** [comment]
Corrections have been made. Kindly review

**Comment by adaugochi** [comment]
Kindly update your PR title to the new task title that is on the app. Going forward, I will have to close this PR so you can then create a new one

**Pull request closed without merging**